Library/Formula/node.rb

```diff
@@ -55,42 +56,50 @@ def install
     system "./configure", *args
     system "make", "install"
 
-    resource("npm").stage libexec/"npm" if build.with? "npm"
-  end
+    if build.with? "npm"
+      resource("npm").stage libexec/"npm_install"
 
-  def post_install
-    return if build.without? "npm"
+      # make sure npm can find node
+      ENV["PATH"] = "#{Formula["node"].prefix}/bin:#{ENV["PATH"]}"
+
+      # set log level temporarily for npm's `make install`
+      ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
+
+      (libexec/"npm_install").cd { system "./configure", "--prefix=#{libexec}/npm" }
+      (libexec/"npm_install").cd { system "make", "install" }
+
+      # FIXTHIS: Manpages aren't currently linking, which is frustrating.
+      rm_rf libexec/"npm/lib/node_modules/npm/share"
```

Comment:
Yeah, if they're not symlinks to the cellar they won't be removed and they'll cause link conflicts. Bummer.

It's tempting to just rm_rf them if we're about to replace them.